src/utils/network_calculations/ybus_calculations.jl

```diff
@@ -186,3 +162,106 @@ function Ybus(sys::System; check_connectivity::Bool = true)
     fixed_admittances = get_components(FixedAdmittance, sys)
     return Ybus(branches, nodes, fixed_admittances; check_connectivity = check_connectivity)
 end
+
+"""
+Nodal incidence matrix (Adjacency) is an N x N matrix describing a power system with N buses. It represents the directed connectivity of the buses in a power system.
+
+The Adjacency Struct is indexed using the Bus Numbers, no need for them to be sequential
+"""
+struct Adjacency{Ax, L <: NTuple{2, Dict}} <: PowerNetworkMatrix{Int}
+    data::SparseArrays.SparseMatrixCSC{Int, Int}
+    axes::Ax
+    lookup::L
+end
+
+"""
+Builds a Adjacency from a collection of buses and branches. The return is an N x N Adjacency Array indexed with the bus numbers.
+
+# Keyword arguments
+- `check_connectivity::Bool`: Checks connectivity of the network using Goderya's algorithm
+"""
+function Adjacency(branches, nodes; check_connectivity::Bool = true)
+    buscount = length(nodes)
+    bus_ax = get_number.(nodes)
+    axes = (bus_ax, bus_ax)
+    bus_lookup = _make_ax_ref(bus_ax)
+    look_up = (bus_lookup, bus_lookup)
+    a = SparseArrays.spzeros(Int, buscount, buscount)
+    for b in branches
```

Comment:
This code is pretty much the same as 

https://github.com/NREL-SIIP/PowerSystems.jl/blob/d83466c5b73058d9d1f3673b7a511248edc82d59/src/utils/network_calculations/ptdf_calculations.jl#L34-L51